tests/1-example-tests.bats

```diff
@@ -0,0 +1,72 @@
+setup() {
+    load 'tests_helper'
+    _tests_helper
+}
+
+
+@test "0: Pre: Create file and dir" {
+  run mkdir testing
+  run touch testing/example
+  assert_success
+}
+
+#bats test_tags=github:true
+@test "1: Testing file existence" {
+  run ls testing/example
```

Comment:
Depending on other tests ist bad.

Reply:
I reduce the number and make all the tests without dependencies between each other. Thanks!